notify-debouncer-refined/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "notify-debouncer-refined"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.56"
+description = "notify event debouncer optimized for ease of use"
+documentation = "https://docs.rs/notify-debouncer-refined"
+homepage = "https://github.com/notify-rs/notify"
+repository = "https://github.com/notify-rs/notify.git"
+authors = ["Daniel Faust <hessijames@gmail.com>"]
+keywords = ["events", "filesystem", "notify", "watch"]
+license = "CC0-1.0 OR Artistic-2.0"
+readme = "README.md"
+
+# See more keys and their definitions at https://doc.rust-lang.org/cargo/reference/manifest.html
+
+[lib]
+name = "notify_debouncer_refined"
+path = "src/lib.rs"
+
+[features]
+default = ["crossbeam"]
+# can't use dep:crossbeam-channel and feature name crossbeam-channel below rust 1.60
```

Comment:
I think we can just break the version compatibility here, if we're doing a major release anyway for notify itself

So feel free to just undo the 1.56 compatibility thingy and I may clean up stuff like this later on

Reply:
I undid the 1.56 compatibility commit and raised the MSRV to 1.60.